android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -4,8 +4,3 @@ distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
 networkTimeout=10000
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-org.gradle.daemon=false
-org.gradle.parallel=true
-org.gradle.jvmargs=-Xmx2560m -XX\:MaxPermSize\=1024m -XX\:+UseParallelGC
-android.useAndroidX=true
-android.enableJetifier=true
```

Comment:
These lines do not belong here, these properties should only be in the root `gradle.properties` file.